rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -768,6 +768,12 @@ rm -f %{_builddir}/file.list.%{name}
 %config(noreplace) /etc/ld.so.conf.d/rudder.conf
 %endif
 
+%config(noreplace) /opt/rudder/etc/uuid.hive
+%config(noreplace) /etc/cron.d/rudder-agent
+%config(noreplace) /etc/init.d/rudder-agent
+%config(noreplace) /etc/default/rudder-agent
+%config(noreplace) /etc/profile.d/rudder-agent.sh
```

Comment:
I don't think we want to mark these as config files on RPM based systems.

The reason they are on deb-based systems is that dpkg will ask the user what to do if a config file is modified, and usually a user will just replace it if he doesn't know what's going on.

However, rpm does not ask questions, and silently places a <file>.rpmnew next to the target file. This results in upgrades not actually upgrading important files - in the list above, the cron and init files (at least) are very important, and need to be updated. This is why we removed them as config files, to make sure they were overwritten.